go/worker/storage/committee/node.go

```diff
@@ -750,6 +770,26 @@ func (n *Node) watchQuit() {
 	close(n.quitCh)
 }
 
+func (n *Node) nudgeAvailability(lastSynced, latest uint64) {
+	if lastSynced == n.undefinedRound || latest == n.undefinedRound {
+		return
+	}
+	if latest-lastSynced < maximumRoundDelayForAvailability {
```

Comment:
Should we also stop being available (e.g., `n.roleProvider.SetUnavailable()`) if we fall too far behind (using a separate threshold to avoid oscillations around the boundary)?

Reply:
Sounds sensible; added it.